src/EnergyPlus/OutputProcessor.cc

```diff
@@ -4430,7 +4430,7 @@ namespace OutputProcessor {
             return OutputProcessor::Unit::V;
         } else if (unitUpper == "KMOL/S") {
             return OutputProcessor::Unit::kmol_s;
-        } else if (unitUpper == "KG/S") {
+        } else if (unitUpper == "REV/MIN") {
```

Comment:
[src/EnergyPlus/OutputProcessor.cc:4433]:(warning),[identicalConditionAfterEarlyExit],Identical condition 'unitUpper=="KG/S"', second condition is always false